deisctl/Makefile

```diff
@@ -32,14 +32,16 @@ install:
 	godep go install -v .
 
 setup-root-gotools:
-	sudo GOPATH=/tmp/tmpGOPATH go get -u -v code.google.com/p/go.tools/cmd/cover
-	sudo GOPATH=/tmp/tmpGOPATH go get -u -v code.google.com/p/go.tools/cmd/vet
+	sudo GOPATH=/tmp/tmpGOPATH go get -u -v golang.org/x/tools/cmd/cover
+	sudo GOPATH=/tmp/tmpGOPATH go get -u -v golang.org/x/tools/cmd/vet
 	sudo rm -rf /tmp/tmpGOPATH
 
 setup-gotools:
-	go get -v github.com/golang/lint/golint
+	go get -u github.com/golang/lint/golint
+	go get -u golang.org/x/tools/cmd/cover
```

Comment:
`setup-gotools` works fine for me on OS X 10.9 with go 1.4.2 installed using brew. I guess it's because the brew folder is user-writeable by default.
```
ls -la /usr/local/opt/go
lrwxr-xr-x  1 johannes  admin  18 Feb 18 17:55 /usr/local/opt/go -> ../Cellar/go/1.4.2
```